decidim-admin/config/locales/ca.yml

```diff
@@ -542,7 +542,7 @@ ca:
             hidden_at: Data d'ocultació
             report_count: Recompte
             reportable: Tipus
-            reported_content: Contingut
+            reported_content_url: Enllaç al contingut
```

Comment:
Don't change locales other than English at GitHub, it causes merge conflicts when pulling from Crowdin.